README.md

```diff
@@ -47,18 +47,17 @@ and add an alias to your shell configuration file: `alias sed=gsed` (e.g. to `~/
 
 Install Android SDK and Build-Tools for API level 30+
 
-> [!important]
```

Comment:
I wish we didn't have to get rid of annotations like this; they are good for readability on github.

What do you think of working around this with a build script? During the build process in `build.rs`, run a sed command or equivalent to remove this kind of thing, storing the result in a generated file in `/tmp`, and then exposing `STRIPPED_README_PATH` in the context of the rest of the build. Then the doc-inclusion becomes

```rust
#![doc = include_str!(env!("STRIPPED_README_PATH"))]

Reply:
Unfortunately, rustdoc doesn't invoke the build script so that won't work. I pushed a version where we add a rustdoc wrapper script and tell Cargo to use that instead of rustdoc directly.

Reply:
I like your fix!